ts/Extensions/DashboardPlugins/HighchartsComponent.ts

```diff
@@ -82,6 +81,9 @@ class HighchartsComponent extends Component<HighchartsComponent.ChartComponentEv
 
     public static syncHandlers = HighchartsSyncHandlers;
```

Comment:
Do we need this property in API? And some others?

![image](https://user-images.githubusercontent.com/1453926/225004223-632d6c0c-9ae2-4957-a2e7-e65b7e1a83d4.png)